recipes/embree3/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "embree3" %}
+{% set version = "3.2.0" %}
+{% set sha256 = "9f4fe73dc2b1fc5ff26840d65d87dcba9b5faef70fccb8c231d85437e0a5bf34" %}
+
+package:
+  name: {{ name }}
+  version: "{{ version }}"
+
+source:
+  fn: {{ version }}.tar.gz
+  url: https://github.com/embree/embree/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  # upstream supports visual studio 13+
+  skip: true  # [win and vc<13]]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+
+  host:
+    - tbb
+
+  run:
+    - tbb
+
+test:
+  commands:
+    - test -f "${PREFIX}/lib/libembree3.${SHLIB_EXT}"  # [not win]
```

Comment:
I think you don't need the `.` here because `${SHLIB_EXT}` starts with a `.`

Reply:
Thanks, that fixed the failing builds.